### PR TITLE
Adding akka-diagnostics

### PR DESCRIPTION
**Pull request by franciscolopezsancho**
References https://github.com/akka/akka-diagnostics/issues/19

### CODE REVIEW

**Comment by patriknw** [comment]
looks like we have some warnings https://github.com/akka/akka-guide/actions/runs/4074859245/jobs/7020504974#step:6:91

**Comment by franciscolopezsancho** [comment]
> https://github.com/akka/akka-guide/actions/runs/4074859245/jobs/7020504974#step:6:91

Please have a look at https://github.com/akka/akka/issues/31830 @patriknw

**Comment by franciscolopezsancho** [comment]
There are no more warnings related to the `ConfigChecker` after using the latest 2.0.0-M4 `akka-diagnostics` and fixing some wrong paths in the `.conf` files.

**Comment by franciscolopezsancho** [comment]
do I have your blessings @patriknw?

File: docs-source/docs/modules/microservices-tutorial/examples/00-shopping-cart-service-scala/src/main/resources/persistence.conf [hunk 1]
```diff
@@ -2,10 +2,10 @@ akka {
   # use JDBC plugin to store both snapshots and the events of the persistent actors
   persistence {
     journal.plugin = "jdbc-journal"
-    auto-start-journals = ["jdbc-journal"]
+    journal.auto-start-journals = ["jdbc-journal"]
```
Comment (patriknw) [inline docs-source/docs/modules/microservices-tutorial/examples/00-shopping-cart-service-scala/src/main/resources/persistence.conf:5]: good catch

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**